app/src/main/java/org/wikipedia/page/edithistory/EditHistoryListActivity.kt

```diff
@@ -90,6 +90,11 @@ class EditHistoryListActivity : BaseActivity() {
                 loadHeader.loadState = it.refresh
                 loadFooter.loadState = it.append
             }
+            viewModel.editHistoryStatsFlow.collectLatest {
```

Comment:
As I understand it, you can't put more than one `collect...` function in a single `launchWhenCreated` call, because the `collect` function never actually returns.
Each `collect` function must be inside its own `launchWhenCreated` scope.

Reply:
Yes, you are correct! Thanks for the PR and tips 👍